fastestimator/op/tensorop/augmentation/cutmix_batch.py

```diff
@@ -98,32 +97,42 @@ def _get_patch_coordinates(tensor: Tensor, x: Tensor, y: Tensor,
         """
         _, img_height, img_width = get_image_dims(tensor)
 
-        cut_x = img_width * x
-        cut_y = img_height * y
-        cut_w = img_width * tensor_sqrt(1 - lam)
-        cut_h = img_height * tensor_sqrt(1 - lam)
+        if tf.is_tensor(tensor):
```

Comment:
is there any reason not to cast for torch also to reduce the if branching?

Reply:
Initially we faced issues in tf, for which we included casting operation. Then while testing the updated code with pytorch, there was a multiplication issue created due to casting (Multiplication of numpy array with torch tensor). SInce casting wasnt necessary for torch, I just converted to if branch